src/parser/tptp/Tptp.g

```diff
@@ -1437,12 +1437,12 @@ tffLetTermBinding[std::vector<CVC4::api::Term> & bvlist,
                   CVC4::api::Term& lhs,
                   CVC4::api::Term& rhs]
   : term[lhs] EQUAL_TOK term[rhs]
-  {
-    PARSER_STATE->checkLetBinding(bvlist, lhs, rhs, false);
-    std::vector<api::Term> lchildren(++lhs.begin(), lhs.end());
-    rhs = MK_TERM(api::LAMBDA, MK_TERM(api::BOUND_VAR_LIST, lchildren), rhs);
-    lhs = api::Term(lhs.getExpr().getOperator());
-  }
+{
```

Comment:
This is very nit-picky but the indentation here is off: the curly brackets should align with the beginning of `term[lhs]` above.

Reply:
Clang-format screwed with this one, did we ever decide what to do to resolve this problem? Switch it off for the .g files?

Reply:
We don't have a good solution.  I think we decided to do our best effort to keep things close to clang-formatted everywhere, but don't require it if its a major effort.